packages/database/src/migrations/20210921051858-LESMISRepititionRatesDistrict-modifies-data.js

```diff
@@ -182,6 +181,27 @@ const getGpiMapOverlay = (name, reportCode) => ({
   projectCodes: '{laos_schools}',
 });
 
+const OVERLAY_GROUPS = [
+  {
+    parentCode: 'Root',
+    children: [
+      {
+        name: 'Repetition Rate',
+        code: 'LESMIS_Repetition_Rate_Group',
+      },
+    ],
+  },
+  {
+    parentCode: 'LESMIS_Repetition_Rate_Group',
+    children: [
+      {
+        name: 'District Level',
+        code: 'LESMIS_Repetition_Rate_District_Group',
+      },
+    ],
+  },
+];
+
```

Comment:
I'm not sure this is correct. Those overlay groups still exist in the database from what I can see:

![image](https://user-images.githubusercontent.com/69437175/135537488-45a5da5a-efc7-4a5e-9579-a65c22fffc06.png)